tasks/main.yml

```diff
@@ -23,21 +23,88 @@
     state: present
   notify: "Restart Fluentbit."
 
-- name: "Create needed directory."
+- name: "Create group td-agent-bit"
+  group:
+    name: td-agent-bit
+    state: present
+    system: True
+
+- name: "Create user with adm group"
+  user:
+    name: td-agent-bit
+    create_home: False
+    group: td-agent-bit
+    state: present
+    groups: adm
```

Comment:
groups should be a list.
I'd add a default list into default/main.yml to be able to override it (we'll probably need to add this users to other groups ..)